### DISCUSSION TITLE
Apple M2 Chip Support?

### DISCUSSION THREAD

**Discussion by aderon2** [apohl79/audiogridder#1249] [Q&A] [answered]
I see on the website that there is an installer for M1 chips.  Now that M2 chips are out, are there any plans for supporting those?  I've been thinking of getting one of the new Mac Studios that utilize the M2 chip and using it as an Audiogridder server.  Just trying to figure out if that's currently supported or not.  Thanks!

**Comment by apohl79** [comment, marked as answer]
M2 chips will work as well. Will change the description on the homepage to silicon. Cheers

**Comment by CbeeCbee** [comment]
Hey there, I just installed the M1 version, but it doesn´t work for some reason.
Tried both plug in installers, but the plugin gets blacklisted inside of Cubase since it doesn´t support the architecture.
What am I doing wrong?
The server 1.2.0.1 seems to work fine so far.
I´m on OS 12.7.1 by the way.
THANK YOU!

**Reply by CbeeCbee** [reply]
and when I use the binaries and place  vsts directly in the destined plugin folder my mac complains "files are damaged and should be trashed".
anyone ?? thanks very much
